UI/csv_import.html

```diff
@@ -2,7 +2,111 @@
 <html>
 <head>
     <link rel="stylesheet" type="text/css" href="css/digiroad2.css">
+    <script type="text/javascript" src="node_modules/jquery/dist/jquery.min.js"></script>
+    <script type="text/javascript" src="node_modules/jquery-migrate/dist/jquery-migrate.min.js"></script>
+    <script type="text/javascript" src="node_modules/multiselect-two-sides/dist/js/multiselect.min.js"></script>
     <script type="text/javascript" src="node_modules/lodash/lodash.js"></script>
+    <script type="text/javascript" src="src/view/genericConfirmPopup.js"></script>
+
+    <script type="text/javascript">
+      getMunicipalities();
+
+      var deleteCheckbox = "input[id='deleteCheckbox']";
+      var municipalities;
+
+      function isToDelete() {
+        if ($(deleteCheckbox).is(':checked')) {
+          $(".municipalities").show();
+          $('.btn.btn-primary.btn-lg').prop('disabled', true);
+        } else {
+          $(".municipalities").hide();
+          $('.btn.btn-primary.btn-lg').prop('disabled', false);
+        }
+      }
+
+      function hasValue() {
```

Comment:
This name should be changed since we are not returning a boolean indicating if it hasValue or not